src/black_primer/primer.json

```diff
@@ -149,17 +149,6 @@
       "long_checkout": false,
       "py_versions": ["all"]
     },
-    "sqlalchemy": {
-      "cli_arguments": [
-        "--experimental-string-processing",
-        "--extend-exclude",
-        "/test/orm/test_relationship_criteria.py"
-      ],
-      "expect_formatting_changes": true,
-      "git_clone_url": "https://github.com/sqlalchemy/sqlalchemy.git",
-      "long_checkout": false,
-      "py_versions": ["all"]
-    },
```

Comment:
For other reviewers sqlalchemy have decided to continue py2 support and py2 only syntax exists.